sdk/servicebus/service-bus/test/internal/connectionString.spec.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { parseServiceBusConnectionString } from "../../src/util/connectionStringUtils";
+import chai from "chai";
+
+const assert = chai.assert;
+
+describe("Connection String", () => {
+  const expectedNamespace = "my.servicebus.windows.net";
+  const expectedEndpoint = `sb://${expectedNamespace}`;
+  const expectedEntityPath = "my-entityPath";
+  const expectedSharedAccessSignature = "my-shared-access-signature";
+  const expectedSharedKey = "my-shared-key";
+  const expectedSharedKeyName = "my-shared-key-name";
+  const dummyErrorMsg = "You should not be seeing me";
+
+  it("Extracts Service Bus properties with only Endpoint", () => {
+    const connectionString = `Endpoint=${expectedEndpoint};`;
+    const connectionStringProperties = parseServiceBusConnectionString(connectionString);
+    assert.equal(connectionStringProperties.endpoint, expectedEndpoint);
+    assert.equal(connectionStringProperties.fullyQualifiedNamespace, expectedNamespace);
+  });
+
+  it("Extracts Service Bus properties with Endpoint & EntityPath", () => {
+    const connectionString = `Endpoint=${expectedEndpoint};EntityPath=${expectedEntityPath}`;
```

Comment:
Question: do you think we should be testing with connection strings that have these properties in a different order? 

My thought is 'no' but I wanted to get your take on it.

Reply:
The occasional puristic in me would say yes, but the pragmatic in me would say no.

Reply:
But I went ahead and added a test case for out of order anyway... https://github.com/Azure/azure-sdk-for-js/pull/11949/commits/548d83ef79c58ad8c37dca021acbeb7b124a4f6c